caravel/model/inquiry.py

```diff
@@ -0,0 +1,34 @@
+from google.appengine.ext import ndb
+from caravel.model import listing
+from caravel.model.moderation import ModeratedMixin
+from caravel.model.temporal import TimeOrderMixin
+from caravel.model.principal import PrincipalMixin
+from caravel.model.side_effects import SideEffectsMixin
+from caravel import utils
+
+from flask import render_template
+
+class _Inquiry(TimeOrderMixin, PrincipalMixin, ModeratedMixin, ndb.Model):
+    message = ndb.StringProperty()
+    listing = ndb.KeyProperty(kind=listing.Listing)
+
+class Inquiry(SideEffectsMixin, _Inquiry):
+    def side_effects(self):
```

Comment:
In this case, this is overriding the function from the SideEffectsMixin. I've added more docs — does that make it clearer? (To someone who hasn't seen this pull request.)